### PR TITLE
Add chain testdata

### PR DESCRIPTION
**Pull request by lk16**
Hello world,

I noticed the tests didn't run on clone, so I added 5 files of the chain to a new testdata folder and used that in the tests.
Maybe it would also be nice to say in the README that tests can be run with `python -m py.test tests/`.

lk16

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ dist
 skepticoin/scmversion.py
 
 __pycache__
+*.code-workspace
```
Comment (sashimi-houdini) [inline .gitignore:12]: Nitpick: please keep developer-specific tastes out of this project's .gitignore and configure them on your local machine

**Comment by sashimi-houdini** [comment]
Looks great, thanks!

2 more things:

* `os` import now no longer required as per `flake8` failing
*  could you remove `-k 'not integration'` from `.github/workflows/python-package.yml` ? It was a workaround but your solution makes it unnecessary.

**Comment by lk16** [comment]
I have addressed all those three things.

**Comment by sashimi-houdini** [comment]
> I have addressed all those three things.

Now just get rid of the rest of the flake8 stuff and this can go in. Congrats & thanks!

**Comment by sashimi-houdini** [comment]
Merged outside of github in ed086793fd8f0489ba0741852e7bafc8a164db54

**Pull request closed without merging**